src/App.css

```diff
@@ -50,8 +55,20 @@ tr {
 .footer {
   padding-top: 20px;
   text-align: center;
+  background-color: rgb(233, 231, 232);
+  font-weight: 700;
+}
+.li {
+  list-style: none;
 }
```

Comment:
This doesn't look correct - `.li` shouldn't be a class. `li` is an element. Either rename the class or use `li`